Resources/Prototypes/Traits/physical.yml

```
@@ -806,24 +806,32 @@
         - type: CritModifier
           critThresholdModifier: -50
 
-#- type: trait Floof disabled because op
-#  id: LightAmplification
-#  category: Physical
-#  points: -4
-#  requirements:
-#    - !type:CharacterJobRequirement
-#      inverted: true
-#      jobs:
-#        - Prisoner # Bionics should be "Confiscated" from long term prisoners.
-#    - !type:CharacterTraitRequirement
-#      traits:
-#        - CyberEyes
-#  functions:
-#    - !type:TraitAddComponent
-#      components:
-#        - type: NightVision
-#
-#- type: trait
+- type: trait
+  id: LightAmplification
+  category: Physical
+  points: -4
```

```suggestion
  points: -5
```

In all honest il say even -5 due to how OP its is.